src/Lifecycle.js

```diff
@@ -128,6 +128,7 @@ function _loginWithToken(queryParams) {
         console.log("Logged in with token");
         setLoggedIn({
             userId: data.user_id,
+            deviceId: data.device_id,
```

Comment:
Presumably `deviceId` will end up being `undefined` if you're using a server that doesn't support giving device IDs? Presumably only fairly recent synapses do? Should we disable crypto in this case?